src/components/MultiButton.tsx

```diff
@@ -71,6 +71,7 @@ const MenuButton = styled(ButtonV2)`
 
 const MenuItems = styled(Content)`
   box-shadow: ${shadows.levitate1};
+  z-index: 200;
```

Comment:
er vel ikke noe standardisert system på plass for z-index?

Reply:
Nei. Jo lavere, jo bedre. Dette kan vi sikkert leve med.